__tests__/intergration.test.js

```diff
@@ -7,6 +7,24 @@ const app = require("../app");
 beforeEach(() => seed(data));
 afterAll(() => db.end());
 
+//__________________ Section: /api _______________//
+describe("GET: /api", () => {
+  test("200: responds with a list of api endpoints available", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body: { api_endpoints } }) => {
```

Comment:
This shouldnt be a nested object. You should have an object with a dynamic key so in this case you shouldnt need the body key, you should have access to api_endpoints directly from the initial de-structuring.